kubernetes/kube-state-metrics-cluster-role.yaml

```diff
@@ -8,6 +8,7 @@ rules:
   - nodes
   - pods
   - resourcequotas
+  - replicationcontrollers
   verbs: ["list", "watch"]
 - apiGroups: ["extensions"]
```

Comment:
Currently we [explicitly use](https://github.com/kubernetes/kube-state-metrics/blob/042a59eb73005b30f506f312f01da7a88988c315/deployment.go#L90) the extensions API. But I wasn't aware of this so thanks for the heads up!